### PR TITLE
feat: add telegram subscription

### PR DESCRIPTION
**Pull request by shoom3301**
1. Get all telegram subscriptions for an account
```
http://localhost:1337/api/tg-subscriptions/0xfb3c7eb936cAA12B5A884d612393969A557d4307
```

2. Add a subscription

```
fetch('http://localhost:1337/api/add-tg-subscription', {method: 'POST', body: JSON.stringify({account: '0xfb3c7eb936cAA12B5A884d612393969A557d4307', data: {
  auth_date: 1715154310,
  first_name: "xxxx",
  hash: "1f2b9a99f5162f9b72b78b85c4a16d44c8f85d0a69eb4a5c50082d3fdb64b541",
  id: 11111111,
  photo_url: "......",
  username: "sssss"
}})})
```

3. Send notifications to telegram

```
http://localhost:1337/api/send-tg-notifications/0xfb3c7eb936cAA12B5A884d612393969A557d4307
```

### CODE REVIEW

File: src/api/telegram-subscription/content-types/telegram-subscription/schema.json [hunk 1]
```diff
@@ -16,17 +17,23 @@
       "required": true,
       "unique": false
     },
-    "login": {
-      "type": "string",
-      "required": true
+    "auth_date": {
```
Comment (anxolin) [inline src/api/telegram-subscription/content-types/telegram-subscription/schema.json:20]: why do we mix camel-case and snakecase?

File: src/api/telegram-subscription/content-types/telegram-subscription/schema.json [hunk 2]
```diff
@@ -16,17 +17,23 @@
...
+      "type": "biginteger"
+    },
+    "first_name": {
```
Comment (anxolin) [inline src/api/telegram-subscription/content-types/telegram-subscription/schema.json:23]: same

File: src/api/telegram-subscription/routes/telegram-subscription.ts [hunk 3]
```diff
@@ -4,4 +4,49 @@
...
+      if (!routes) routes = innerRouter.routes.concat(extraRoutes);
+      return routes;
+    },
+  };
+};
+
+const myExtraRoutes = [
+  {
+    method: 'GET',
+    path: '/tg-subscriptions/:account',
```
Comment (anxolin) [inline src/api/telegram-subscription/routes/telegram-subscription.ts:25]: the name is a bit weird. Why not something like:
- `/accounts/{account}/subscriptions?channel=telegram`

**Review by anxolin — COMMENTED** [review]
Nice job figuring out the hmac thing! not making too many comments as this will be cleaned up after hackathon, so all good!

File: src/api/telegram-subscription/content-types/telegram-subscription/schema.json (hunk 1 above)
Comment (shoom3301) [reply src/api/telegram-subscription/content-types/telegram-subscription/schema.json:20]: Initially I left snake_case to be consistent with data format from Telegram API.
But we map the data anyway, changed to camelCase on our side

File: src/api/telegram-subscription/routes/telegram-subscription.ts (hunk 3 above)
Comment (shoom3301) [reply src/api/telegram-subscription/routes/telegram-subscription.ts:25]: Fixed

**Pull request merged**